src/main/steps/applicant/yourdocuments/alldocuments/alldocuments/tasklistalldocuments.ts

```diff
@@ -333,6 +320,83 @@ export const getRespondentDocuments = (sectionTitles, taskListItems, userCase, i
   };
 };
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const isPreviousOrdersSubmittedRespondent = (taskListItems, url, isPreviousOrdersSubmitted): any => {
+  if (isPreviousOrdersSubmitted) {
+    return {
+      id: 'previous_orders_submitted_respondent',
+      text: taskListItems.previous_orders_submitted_respondent,
+      href: url + URL.PREVIOUS_ORDERS_SUBMITTED + '?byApplicant=No',
+    };
+  }
+};
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const isLettersFromSchoolRespondent = (taskListItems, url, isLettersFromSchool): any => {
+  if (isLettersFromSchool) {
+    return {
+      id: 'letters_from_school_respondent',
+      text: taskListItems.letters_from_school_respondent,
+      href: url + URL.LETTER_FROM_SCHOOL + '?byApplicant=No',
+    };
+  }
+};
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const isDigitalDownloadsUploadedRespondent = (taskListItems, url, isDigitalDownloadsUploaded): any => {
+  if (isDigitalDownloadsUploaded) {
+    return {
+      id: 'digital_downloads_respondent',
+      text: taskListItems.digital_downloads_respondent,
+      href: url + URL.DIGITAL_DOWNLOADS + '?byApplicant=No',
+    };
+  }
+};
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const isMedicalRecordsUploadRespondent = (taskListItems, url, isMedicalRecordsUpload): any => {
+  if (isMedicalRecordsUpload) {
+    return {
+      id: 'medical_records_respondent',
+      text: taskListItems.medical_records_respondent,
+      href: url + URL.MEDICAL_RECORDS + '?byApplicant=No',
+    };
+  }
+};
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const isMedicalReportsUploadedRespondent = (taskListItems, url, isMedicalReportsUploaded): any => {
+  if (isMedicalReportsUploaded) {
+    return {
+      id: 'medical_reports_respondent',
+      text: taskListItems.medical_reports_respondent,
+      href: url + URL.MEDICAL_REPORTS + '?byApplicant=No',
+    };
+  }
+};
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const isPaternityDocUploadedRespondent = (taskListItems, url, isPaternityDocUploaded): any => {
```

Comment:
Are there test cases written to cover these individual methods?

Reply:
test cases are written